src/main/java/mate/jdbc/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,121 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.lib.DataProcessingException;
+import mate.jdbc.models.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+
+    @Override
+    public List<Manufacturer> getAll() {
+        List<Manufacturer> allFormat = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                 Statement getAllFormatsStatement = connection.createStatement()) {
+            ResultSet resultSet = getAllFormatsStatement
+                    .executeQuery("SELECT * FROM taxi_db.manufacturers WHERE is_deleted = FALSE");
```

Comment:
```suggestion
                    .executeQuery("SELECT * FROM manufacturers WHERE is_deleted = FALSE");
```
ім'я схеми не треба вказувати, тому що ім'я схеми ти вказуєш в JDBC url строці в ConnectionUtil класі